### PR TITLE
Test with rails 5.1 and ruby 2.4.0

### PR DESCRIPTION
**Pull request by mfazekas**
Updated travis.yml with rails 5.1 and ruby 2.4

### CODE REVIEW

**Comment by BenMorganIO** [comment]
@mfazekas awesome! Are you able to remove Rails 4.1 actually? We drop support as new rails versions come out :). (Keeps the CI fast and makes adding new features or fixing bugs easier for everyone.)

**Comment by mfazekas** [comment]
I've removed 4.1 from travis. (In case you actually break 4.1 you might want to increase major version because of SemVer)

File: .travis.yml [hunk 1]
```diff
@@ -5,22 +5,27 @@ rvm:
   - 2.1.10
```
Comment (BenMorganIO) [inline .travis.yml]: Man, I remember the day when I was super excited for Ruby 2.1, but nowadays I think we can even remove this from Travis as well. 2.2 is quite compatible and gives that restriction from the `?.` operator.

File: .travis.yml [hunk 2]
```diff
@@ -5,22 +5,27 @@ rvm:
...
   - 2.2.6
   - 2.3.3
+  - 2.4.0
```
Comment (BenMorganIO) [inline .travis.yml]: I don't want to keep asking you to keep going back and making changes to something that should be quite simple... But, it looks like the latest is 2.4.1 :p

**Comment by BenMorganIO** [comment]
Merged. I just realized, your commit is a lie now :p lol

**Comment by aried3r** [comment]
Why is Rails 5.1.0 not being tested with Ruby 2.2?

**Comment by mfazekas** [comment]
@aried3r i guess it was an overlook

**Pull request merged**